components/raftstore/src/store/peer.rs

```diff
@@ -2600,11 +2600,12 @@ impl Peer {
     }
 
     pub fn bcast_wake_up_message<T: Transport, C>(&self, ctx: &mut PollContext<T, C>) {
-        ctx.need_flush_trans = true;
         for peer in self.region().get_peers() {
             if peer.get_id() == self.peer_id() {
                 continue;
             }
+            ctx.need_flush_trans = true;
```

Comment:
I think it should be set only send success.

Reply:
Done.